app-backup/amanda/amanda-3.5.1-r3.ebuild

```diff
@@ -188,145 +186,170 @@ src_configure() {
-	myconf="${myconf} --with-index-server=${AMANDA_SERVER_INDEX}"
+	myconf+=( --with-index-server="${AMANDA_SERVER_INDEX}" )
 	einfo "Using ${AMANDA_USER_NAME} for amanda user."
-	myconf="${myconf} --with-user=${AMANDA_USER_NAME}"
+	myconf+=( --with-user="${AMANDA_USER_NAME}" )
 	einfo "Using ${AMANDA_GROUP_NAME} for amanda group."
-	myconf="${myconf} --with-group=${AMANDA_GROUP_NAME}"
+	myconf+=( --with-group="${AMANDA_GROUP_NAME}" )
 	einfo "Using ${AMANDA_TAR} as Tar implementation."
-	myconf="${myconf} --with-gnutar=${AMANDA_TAR}"
+	myconf+=( --with-gnutar="${AMANDA_TAR}" )
 	einfo "Using ${AMANDA_TAR_LISTDIR} as tar listdir."
-	myconf="${myconf} --with-gnutar-listdir=${AMANDA_TAR_LISTDIR}"
+	myconf+=( --with-gnutar-listdir="${AMANDA_TAR_LISTDIR}" )
 	einfo "Using ${AMANDA_CONFIG_NAME} as default config name."
-	myconf="${myconf} --with-config=${AMANDA_CONFIG_NAME}"
+	myconf+=( --with-config="${AMANDA_CONFIG_NAME}" )
 	einfo "Using ${AMANDA_TMPDIR} as Amanda temporary directory."
-	myconf="${myconf} --with-tmpdir=${AMANDA_TMPDIR}"
+	myconf+=( --with-tmpdir="${AMANDA_TMPDIR}" )
 
-	if [ -n "${AMANDA_PORTS_UDP}" ] && [ -n "${AMANDA_PORTS_TCP}" ] && [ -z "${AMANDA_PORTS_BOTH}" ] ; then
+	if [[ -n ${AMANDA_PORTS_UDP} && -n ${AMANDA_PORTS_TCP} && -z ${AMANDA_PORTS_BOTH} ]]; then
 		eerror "If you want _both_ UDP and TCP ports, please use only the"
 		eerror "AMANDA_PORTS environment variable for identical ports, or set"
 		eerror "AMANDA_PORTS_BOTH."
 		die "Bad port setup!"
 	fi
-	if [ -n "${AMANDA_PORTS_UDP}" ]; then
+	if [[ -n ${AMANDA_PORTS_UDP} ]]; then
 		einfo "Using UDP ports ${AMANDA_PORTS_UDP/,/-}"
-		myconf="${myconf} --with-udpportrange=${AMANDA_PORTS_UDP}"
+		myconf+=( --with-udpportrange="${AMANDA_PORTS_UDP}" )
 	fi
-	if [ -n "${AMANDA_PORTS_TCP}" ]; then
+	if [[ -n ${AMANDA_PORTS_TCP} ]]; then
 		einfo "Using TCP ports ${AMANDA_PORTS_TCP/,/-}"
-		myconf="${myconf} --with-tcpportrange=${AMANDA_PORTS_TCP}"
+		myconf+=( --with-tcpportrange="${AMANDA_PORTS_TCP}" )
 	fi
-	if [ -n "${AMANDA_PORTS}" ]; then
+	if [[ -n ${AMANDA_PORTS} ]]; then
 		einfo "Using ports ${AMANDA_PORTS/,/-}"
-		myconf="${myconf} --with-portrange=${AMANDA_PORTS}"
+		myconf+=( --with-portrange="${AMANDA_PORTS}" )
 	fi
 
 	# Extras
 	# Speed option
-	myconf="${myconf} --with-buffered-dump"
+	myconf+=( --with-buffered-dump )
 	# "debugging" in the configuration is NOT debug in the conventional sense.
 	# It is actually just useful output in the application, and should remain
 	# enabled. There are some cases of breakage with MTX tape changers as of
 	# 2.5.1p2 that it exposes when turned off as well.
-	myconf="${myconf} --with-debugging"
+	myconf+=( --with-debugging )
 	# Where to put our files
-	myconf="${myconf} --localstatedir=${AMANDA_USER_HOMEDIR}"
+	myconf+=( --localstatedir="${AMANDA_USER_HOMEDIR}" )
 
 	# Samba support
-	myconf="${myconf} $(use_with samba smbclient /usr/bin/smbclient)"
+	myconf+=( $(use_with samba smbclient /usr/bin/smbclient) )
```

Comment:
I'm not sure we care enough to figure out if the build system prefixes these as needed tbh.

Reply:
agreed